packages/p2p/src/stores/buy-sell-store.js

```diff
@@ -260,6 +266,33 @@ export default class BuySellStore extends BaseStore {
         this.setIsSortDropdownOpen(false);
     }
 
+    handleResponse = async order => {
+        const { sendbird_store, order_store } = this.root_store;
+        if (order.error) {
+            this.form_props.setErrorMessage(order.error.message);
+            this.setFormErrorCode(order.error.code);
+        } else {
+            if (order?.subscription?.id && !this.is_create_order_subscribed) {
+                this.setIsCreateOrderSubscribed(true);
+            }
+            this.form_props.setErrorMessage(null);
+            this.root_store.general_store.hideModal();
+            this.root_store.floating_rate_store.setIsMarketRateChanged(false);
```

Comment:
instead of calling this.root_store, could we add those stores the where you had deconstructured the other stores as well?
```suggestion
            general_store.hideModal();
            floating_rate_store.setIsMarketRateChanged(false);
```